miniamazon-frontend/src/components/Account.js

```diff
@@ -2,17 +2,10 @@ import React from 'react';
 import '../App.css';
 import axios from 'axios';
 
-var userID = "compsci316project";
```

Comment:
I'm getting `___ is not defined` errors for all of these variables since they are used in 44-51

Reply:
me too. I am looking into it

Reply:
I think you have to retrieve the current user, but honestly that can be done after Martin finishes log in/sign up so you can just add these lines back

Reply:
fixed! and I think ready w/ last commit